src/wp-includes/author-template.php

```diff
@@ -85,11 +85,19 @@ function the_author( $deprecated = '', $deprecated_echo = true ) {
  * Retrieves the author who last edited the current post.
  *
  * @since 2.8.0
+ * @since 6.2.0 Added $post parameter.
  *
+ * @param int|WP_Post $post Post ID or post object. Default is global `$post` object.
```

Comment:
Similar to https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/blocks.php#L563 should update the doc.

```suggestion
 * @param int|WP_Post|null $post Optional. Post ID or post object. Default is global `$post` object.
```